default/stringtables/en.txt

```diff
@@ -11374,6 +11391,9 @@ GREAT_DEFENSE_TROOPS
 ULTIMATE_DEFENSE_TROOPS
 +++	Ultimate Defensive Ground [[encyclopedia TROOP_TITLE]]: 300%
 
+ANCIENT_DEFENSE_TROOPS
++++	Ancient Defensive Ground [[encyclopedia TROOP_TITLE]]: 10 per population
+
```

Comment:
this and the below shield are where the merge conflicts were for me, I made a hash of sorting it out so better to be redone, both need the _desc suffix as above.